### ISSUE TITLE
Executing a DROP TABLE for a table that does not exist returns message with invalid characters.

### ISSUE THREAD

**Issue by chrhol** [Starcounter/Home#206]
Trying to drop a table that does not exist returns an errormessage containing invalid characters. Tested in the latest versions in both `v2.3` and `v2.4 (develop)`, no exact version since they are built locally.

```
ScErrTypeNotFound (SCERR15003): There is no type in the database for the given type name. Table not found The error near or at position -1, near or at token: ☐☐☐☐☐☐☐☐☐☐☐☐☐☐☐☐☐☐
In query: DROP TABLE NotExisting
Version: 2.0.0.
Help page: https://github.com/Starcounter/Starcounter/wiki/SCERR15003.
```

**How to reproduce**

Simply do a call to drop a table that does not exist. If done from Administrator the actual message will be hidden in an exception raised from trying to serialize the invalid string.

```csharp
class Program {
    static void Main() {
        Db.SlowSQL("DROP TABLE NotExisting");
    }
}
```

**Comment by chrhol** [comment]
Copied from other issue
 
> @alexanderstc:
> Metalayer seems to be reporting a correct text error message like "Table NoSuchName not found."

Could it be that the codehost reads too much from the buffer from the metalayer so parts of old errormessages gets added? It looks a bit mangled, `Table not found The error near or at position -1`.

I will take a look.

**Comment by alexanderstc** [comment]
There is also query processor in between, probably.

**Comment by alexanderstc** [comment]
Lemme try some QP test.

**Comment by k-rus** [comment]
cc @andan342

**Comment by alexanderstc** [comment]
@andan342 looks like the token element of the scerror structure is not null and points to garbage.

**Comment by k-rus** [comment]
The issue should be fixed.

**Comment by k-rus** [comment]
I experienced this problem in 2.3 now. I see that the problem was not fixed in 2.3, while the OP states both 2.3 and develop.

@andan342 this needs to be fixed urgently as the fixed needs to be included into 2.3.1 release (cc @miyconst )
